### DISCUSSION TITLE
V 1.0 release timeline

### DISCUSSION THREAD

**Discussion by 3kwa** [osqp/osqp#596] [General]
Just wondering if there is a timeline for the v 1.0 release. Things seem to have stalled so wondering if help should be provided?

**Comment by bstellato** [comment]
Thanks for reaching out @3kwa. Progress has been slower than expected but we are getting closer. We have been fixing some issues with the interfaces (e.g., https://github.com/osqp/osqp-matlab/pull/52), thanks to the great help of @imciner2 and @AmitSolomonPrinceton. I believe we should be quite close (mostly working on interfaces, [migration guide](https://github.com/osqp/osqp/pull/593), etc.). We should be able to release OSQP early next year.

**Reply by 3kwa** [reply]
Hi Bartolomeo, any update ? Extending again an offer to help if need be.

**Reply by FranekStark** [reply]
Hi @bstellato any update on this? :)

**Comment by kgreenek** [comment]
I thought I'd give this a bump. I've been using the latest commit on HEAD for quite a while. It'll reduce a bit of maintenance burden for me once v1.0 is officially released. In particular, I'd like to update the [upstream bazel central registry](https://github.com/bazelbuild/bazel-central-registry/tree/main/modules/osqp) to v1.0 and I can do that same for [osqp-eigen](https://github.com/robotology/osqp-eigen) when they publish an update after that.

**Comment by vineetbansal** [comment]
We've (finally) tagged a commit from a few days ago as version 1.0.0. The documentation is updated, and the release artifacts should be on the `Releases` page on github. We've also released python wheels for it on PyPI. Check it out!
